src/lib/chat/matrix-client.ts

```diff
@@ -42,6 +42,7 @@ import { encryptFile } from './matrix/media';
 import { uploadAttachment } from '../../store/messages/api';
 import { featureFlags } from '../feature-flags';
 import { logger } from 'matrix-js-sdk/lib/logger';
+import { USER_TYPING_TIMEOUT } from '../../store/channels/saga';
```

Comment:
Probably best not to import saga information here. The matrix client really shouldn't "know" things about the rest of the system.